test/java/seedu/addressbook/commands/DeleteCommandTest.java

```diff
@@ -33,13 +33,13 @@
     @Before
     public void setUp() throws Exception {
         Person johnDoe = new Person(new Name("John Doe"), new Phone("61234567", false),
-                new Email("john@doe.com", false), new Address("395C Ben Road", false), new UniqueTagList());
+                new Email("john@doe.com", false), new Address("395C, Ben Road, #01-01, 132324", false), new UniqueTagList());
```

Comment:
[Coding standard](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html) violation! 
If the line has to be split into multiple lines, try to align the parameters.

E.g.
``` 
Person johnDoe = new Person(new Name("John Doe"), 
                 new Phone("61234567", false),
                 ...
```